### PR TITLE
add libraries google-cloud-bom is dropping

### PR DESCRIPTION
**Pull request by elharo**
@suztomo @stephaniewang526

### CODE REVIEW

**Comment by suztomo** [comment]
Strange. I don't see com.google.cloud:google-cloud-storage in this BOM.

```
[ERROR]     'dependencies.dependency.version' for com.google.cloud:google-cloud-core:jar is missing. @ line 57, column 17
[ERROR]     'dependencies.dependency.version' for com.google.cloud:google-cloud-storage:jar is missing. @ line 61, column 17
```

File: boms/cloud-oss-bom/pom.xml [hunk 1]
```diff
@@ -49,6 +49,9 @@
     <io.grpc.version>1.32.1</io.grpc.version>
     <protobuf.version>3.13.0</protobuf.version>
     <http.version>1.36.0</http.version>
+    <gax.version>1.59.0</gax.version>
+    <common.protos.version>1.18.1</protos.version>
```
Comment (suztomo) [inline boms/cloud-oss-bom/pom.xml:53]: Found the closing tag mismatch.

```
Caused by: org.codehaus.plexus.util.xml.pull.XmlPullParserException: end tag name </protos.version> must match start tag name <common.protos.version> from line 53 (position: TEXT seen ...<common.protos.version>1.18.1</protos.version>... @53:51) 
```

**Review by suztomo — APPROVED** [review]

**Comment by suztomo** [comment]
LGTM. I checked the effective pom and it showed only the order of the entry changed:

```
suztomo-macbookpro44% mvn help:effective-pom  |grep artifactId > /tmp/pom-master.xml
suztomo-macbookpro44% git diff /tmp/pom-master.xml /tmp/pom-all.xml 
diff --git a/tmp/pom-master.xml b/tmp/pom-all.xml
index bc20680b..112c08cf 100644
--- a/tmp/pom-master.xml
+++ b/tmp/pom-all.xml
@@ -21,15 +21,15 @@
         <artifactId>grpc-services</artifactId>
         <artifactId>grpc-stub</artifactId>
         <artifactId>grpc-testing</artifactId>
-        <artifactId>guava</artifactId>
-        <artifactId>guava-testlib</artifactId>
-        <artifactId>protobuf-java</artifactId>
-        <artifactId>protobuf-java-util</artifactId>
         <artifactId>api-common</artifactId>
         <artifactId>proto-google-common-protos</artifactId>
         <artifactId>grpc-google-common-protos</artifactId>
         <artifactId>proto-google-iam-v1</artifactId>
         <artifactId>grpc-google-iam-v1</artifactId>
+        <artifactId>guava</artifactId>
+        <artifactId>guava-testlib</artifactId>
+        <artifactId>protobuf-java</artifactId>
+        <artifactId>protobuf-java-util</artifactId>
         <artifactId>google-cloud-bigquery</artifactId>
         <artifactId>google-cloud-compute</artifactId>
         <artifactId>google-cloud-dns</artifactId>
```

**Comment by elharo** [comment]
Not surprising that the order changed. I don't think the order matters here since this is dependencyManagement, not dependencies.

**Pull request merged**